src/main/java/emu/grasscutter/server/packet/send/PacketSceneTeamUpdateNotify.java

```diff
@@ -24,14 +25,19 @@ public PacketSceneTeamUpdateNotify(Player player) {
 						.setSceneId(p.getSceneId())
 						.setEntityId(entityAvatar.getId())
 						.setSceneEntityInfo(entityAvatar.toProto())
-						.setWeaponGuid(entityAvatar.getAvatar().getWeapon().getGuid())
+//						.setWeaponGuid(entityAvatar.getAvatar().getWeapon().getGuid())
 						.setWeaponEntityId(entityAvatar.getWeaponEntityId())
 						.setIsPlayerCurAvatar(p.getTeamManager().getCurrentAvatarEntity() == entityAvatar)
 						.setIsOnScene(p.getTeamManager().getCurrentAvatarEntity() == entityAvatar)
 						.setAvatarAbilityInfo(AbilitySyncStateInfo.newBuilder())
 						.setWeaponAbilityInfo(AbilitySyncStateInfo.newBuilder())
 						.setAbilityControlBlock(entityAvatar.getAbilityControlBlock());
-				
+				GameItem weapon = entityAvatar.getAvatar().getWeapon();
```

Comment:
Please explain this change too. Under what circumstances does this `crash` occur and under what circumstances would avatar weapon be `null`?